### ISSUE TITLE
Error window when starting Mesen from the command line with fullscreen parameter

### ISSUE THREAD

**Issue by nicolasldc** [SourMesen/Mesen#797]
![mesen error message](https://user-images.githubusercontent.com/62917998/77976987-d3c8b200-72d4-11ea-9c46-1817f4bb0e70.jpg)

When launching Mesen from a frontend with the commandline below I get the attached error message. After clicking ok emulation starts without further issues. If I remove the /fullscreen command no error message is presented. I tested it on CMD too just to make sure it was not something funny happening with the forntend shell command - result from the command line is identical.

command line: Mesen.exe /fullscreen "G:\Emulation\Frontend\Hikari\Roms\Nintendo - Nintendo Entertainment System\1943 - The Battle of Midway (USA).zip"

PC specs: AMD Ryzen 3900X, Nvidia 2070 super with latest drivers, asus MB

**Comment by SourMesen** [comment]
Thanks for the report. I can't reproduce this on my end unfortunately.

I understand what the error you've posted is saying, but I can't understand how that particular situation could happen (or how the /fullscreen flag in particular could affect it..)
Can you try to see if you get the same error with the default settings, to ensure it's not something that's caused by some option? (e.g rename your settings.xml file to something else to make a backup of it and force all settings to their default values)

**Comment by nicolasldc** [comment]
Hi, good morning!
I had also tested on a clean new download but got the same error. I
concluded it was the fullscreen command because I tested starting Mesen
from cmd and starting Mesen and launching a game without the /fullscreen
and it did not display the error. Having said that, after restarting the
computer because I got errors on other applications the problem disapeared.
So I believe the issue was on my computer somehow, sorry for disturbin.
Thus far I'm also unable to reproduce the error. Thanks for the amazing
emulator!!

Em Seg, 30 de mar de 2020 22:48, SourMesen <notifications@github.com>
escreveu:
